docs/changelogs/v0.33.md

```diff
@@ -14,11 +14,22 @@
 
 ### 🔦 Highlights
 
+#### Bitswap improvements from Boxo
+
+This release includes some refactorings and improvements affecting Bitswap which should improve reliability. One of the changes affects blocks providing. Previously, the bitswap layer took care itself of announcing new blocks -added or received- with the configured provider (i.e. DHT). This bypassed the "Reprovider", that is, the system that manages precisely "providing" the blocks stored by Kubo. The Reprovider knows how to take advantage of the [AcceleratedDHTClient](https://github.com/ipfs/kubo/blob/master/docs/config.md#routingaccelerateddhtclient), is able to handle priorities, logs statistics and is able to resume on daemon reboot where it left off. From now on, Bitswap will not be doing any providing on-the-side and all announcements are managed by the reprovider. In some cases, when the reproviding queue is full with other elements, this may cause additional delays, but more likely this will result in improved block-providing behaviour overall.
+
 #### 📦️ Dependency updates
 
 - update `boxo` to [v0.24.TODO](https://github.com/ipfs/boxo/releases/tag/v0.24.TODO)
 - update `go-libp2p` to [v0.37.1](https://github.com/libp2p/go-libp2p/releases/tag/v0.37.1) + [v0.37.2](https://github.com/libp2p/go-libp2p/releases/tag/v0.37.2)
 
 ### 📝 Changelog
 
+<details><summary>Full Changelog</summary>
+
+- github.com/ipfs/kubo
+  - Fix a deadlock issue in the Pinner when GC is cancelled  ([ipfs/kubo#10593](https://github.com/ipfs/kubo/issues/10593))
+
+</details>
+
```

Comment:
nit: we don't touch this section in PRs, it is generated during release ([docs](https://github.com/ipfs/kubo/blob/master/docs/RELEASE_CHECKLIST.md#the-release)):
```suggestion
```